### DISCUSSION TITLE
[Blockchain request]: Pulsechain

### DISCUSSION THREAD

**Discussion by reystar22** [LedgerHQ/ledger-live-desktop#4892] [Coin/Token request]
@ledgerlive 

Please consider adding PulseChain (https://pulsechain.com/) to Ledger Live.

It's an Ethereum Hard fork and its MASSIVE.

There was a sacrifice phase that the total volume that was sacrificed for Pulsechain was around $700 Million 

Pulsechain also launching their DEX (Uniswap equivelant) called PulseX. There was also a sacrifice phase for PulseX with over 1 BILLION $ was sacrificed!!

**Trezor has already added support for Pulsechain, hope you will add support for Pulsechain and all it's tokens ASAP.**

**Comment by tester2011** [comment]
Yes, I agree, would be a nice addition...
